docs/index.md

```diff
@@ -4,23 +4,37 @@ hide:
+Event-by-event hybrid model for the description of relativistic heavy-ion collisions in the low and high baryon-density regime.
+This model constitutes a chain of different submodules to appropriately describe each phase of the collision with its corresponding degrees of freedom.
+It consists of the following stages and supported software modules are listed within each stage:
 
-:cloud_tornado: &nbsp; **SMASH** hadronic transport approach to provide the initial conditions
+:cloud_tornado: &nbsp; **Initial conditions**
 
-:droplet: &nbsp; **vHLLE** 3+1D viscous hydrodynamics approach to describe the evolution of the hot and dense fireball
+:   :simple-ticktick: &nbsp; *SMASH* (hadronic transport approach)
 
-:   :material-arrow-right-bottom: &nbsp; CORNELIUS tool to construct a hypersurface of constant energy density from the hydrodynamical evolution (embedded in vHLLE)
+:droplet: &nbsp; **Hydrodynamics** to describe the evolution of the hot and dense fireball
 
-:seedling: &nbsp; **Sampler** to perform Cooper-Frye particlization of the elements on the freezeout hypersurface
+:   :simple-ticktick: &nbsp; *vHLLE* (3+1D viscous hydrodynamics approach)
+    :   :material-arrow-right-bottom: &nbsp; including *CORNELIUS* to construct a constant energy density hypersurface from the hydrodynamical evolution
 
-:fire: &nbsp; **SMASH** hadronic transport approach to perform the afterburner evolution
+:seedling: &nbsp; **Hadron sampler** to perform Cooper-Frye particlization of the elements on the freezeout hypersurface
 
-!!! info "Give credit appropriately"
+:   :simple-ticktick: &nbsp; *SMASH hadron sampler*
+:   :simple-ticktick: &nbsp; *FIST sampler*
 
-    If you are using the SMASH-vHLLE-hybrid, please cite [Eur.Phys.J.A 58(2022)11,230](https://arxiv.org/abs/2112.08724).
-    You may also consult this reference for further details about the hybrid approach.
+:fire: &nbsp; **Afterburner** evolution
+
+:   :simple-ticktick: &nbsp; *SMASH* (hadronic transport approach)
+
+!!! info "Give credit appropriately &nbsp; [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.15880337.svg){ align=right }](https://doi.org/10.5281/zenodo.15880337)"
+
+    If you are using the Hybrid-handler, please cite the corresponding [:simple-doi: software DOI](https://doi.org/10.5281/zenodo.15880337) of the specific code version employed.
+    Depending on the stages and modules you are using, please cite the appropriate papers and software DOIs as well.
+
+    For example, if you use the SMASH-vHLLE-Hybrid, besides citing the Hybrid-handler itself, you should cite [:newspaper: Eur.Phys.J.A 58(2022)11, 230](https://arxiv.org/abs/2112.08724) for the general approach.
```

Comment:
I would rather change this to something like that makes it clear what `SMASH-vHLLE-Hybrid` means. 🤔 I would then remove _about the SMASH-vHLLE-Hybrid approach_ in the following sentence.



```suggestion
    For example, if you use the SMASH-vHLLE-Hybrid approach, besides citing the Hybrid-handler itself, you should cite [:newspaper: Eur.Phys.J.A 58(2022)11, 230](https://arxiv.org/abs/2112.08724) for physical aspects.
```